client/packages/system/src/Stock/Components/StockLineEditModal.tsx

```diff
@@ -60,10 +62,19 @@ export const StockLineEditModal: FC<StockLineEditModalProps> = ({
   });
 
   const { draft, onUpdate, onSave } = useDraftStockLine(stockLine);
+  const { dispatchEvent, addEventListener, removeEventListener } =
+    usePluginEvents();
+  const [hasChanged, setHasChanged] = useState(false);
+  const plugins = usePluginElements({
```

Comment:
pulled the plugins up here, because the form is mutating the `draft` object on every keypress. the plugin, I decided, doesn't need to have the current values, and is ok with the original value of the stockline object